mod-source-record-manager-server/src/test/java/org/folio/rest/impl/metadataProvider/MetaDataProviderJobLogEntriesAPITest.java

```diff
@@ -1089,6 +1089,45 @@ public void shouldReturnDiscardedForMarcBibAndInstanceIfMarcBibNotMatched(TestCo
     }));
   }
 
+  @Test
+  public void shouldReturnDiscardedForAuthorityIfAuthorityMatchedAndErrorAndOtherAuthorityUpdated(TestContext context) {
```

Comment:
@RomanChernetskyi  Maybe it would be better to rename "shouldReturnDiscardedForAuthorityIfAuthorityMatched**With**ErrorAndOtherAuthorityUpdated" ?